Common/Python/PythonData.cs

```diff
@@ -42,6 +43,13 @@ public PythonData()
         public PythonData(PyObject pythonData)
         {
             _pythonData = pythonData;
+            using (Py.GIL())
+            {
+                if (pythonData.HasAttr("UsesMapFiles"))
+                {
+                    _usesMapFiles = _pythonData.UsesMapFiles();
```

Comment:
very cool -- I assume we're only supporting `true`/`false` constants here. Just wondering if it _should_ be a method that gets passed `self` so it could check it's symbol or similar?

Reply:
Exactly - It will be passed `self`, check out the custom data defined at `CustomDataUsingMapFileRegressionAlgorithm.py`